MAINTAINER_GUIDELINES.md

```diff
@@ -0,0 +1,11 @@
+## Maintainer Guidelines
+
+### Releasing working code
+
+* run a full suite of actual integration tests before accepting a PR
+* releases should have a full integration test run before releasing to catch stragglers
+
+### Development
+
+* Rusoto should be able to be used and developed on Rust's stable channel
+* to hit Rusoto's 1.0 release we should have all known AWS related bugs fixed (Ceph and minio are not required for 1.0) and all services covered
```

Comment:
Should we perhaps also add a comment about requiring integration tests for all services?

Reply:
That's a good idea!  We're lacking some integration tests because they are tricky to implement. It's hard to say we're 1.0 and API stable if we can't demonstrate we can interact with all services. 😄